Sources/DolbyIORTSUIKit/Public/Screens/Settings/SettingsScreen.swift

```diff
@@ -8,14 +8,14 @@ import DolbyIOUIKit
 public struct SettingsScreen: View {
 
     @Environment(\.presentationMode) var presentationMode
-    @StateObject private var viewModel: SettingsViewModel = .init()
+    @StateObject private var viewModel: SettingsViewModel
 
     @State private var isShowingMultiviewScreen: Bool = false
     @State private var isShowingStreamSortOrderScreen: Bool = false
     @State private var isShowingAudioSelectionScreen: Bool = false
 
-    public init() {
-
+    public init(mode: SettingsMode) {
```

Comment:
`public init(mode: SettingsMode = .global)` 
Global shall be the most commonly used case, and a user doesn't have to explicitly know this global if he/she uses low code.

Reply:
I am 100% sure with assigning a default value for the arguments of this type is helpful - as it may leave the user with a global settings experience if the user does not specifically look into the init method. 

However, I will hold on this change for now and proceed after we discuss on the design with @RameshDPrasad tomorrow.